### DISCUSSION TITLE
next  assetPrefix: How to implement dynamic functions that only support strings, as my CDN can have multiple addresses and I want to distribute them across different CDNs

### DISCUSSION THREAD

**Discussion by parlay96** [vercel/next.js#69250] [Help]
### Summary

next  assetPrefix: How to implement dynamic functions that only support strings, as my CDN can have multiple addresses and I want to distribute them across different CDNs  

### Additional information

```js
next  assetPrefix: How to implement dynamic functions that only support strings, as my CDN can have multiple addresses and I want to distribute them across different CDNs
```

**Comment by parlay96** [comment]
![image](https://github.com/user-attachments/assets/8e22468a-f307-4469-8a0e-2413ceb46ee2)

**Comment by parlay96** [comment]
Does anyone know? Can you tell me? Thank you

**Comment by parlay96** [comment]
How to configure multiple CDN addresses？

**Comment by parlay96** [comment]
How to configure multiple CDN addresses？

**Comment by ZoranRavic** [comment]
There is no option for dynamic asset prefix.

You'll have to write custom logic for this.

**Reply by parlay96** [reply]
> There is no option for dynamic asset prefix.
> 
> You'll have to write custom logic for this.

Thank you, do you know how to achieve it?
